source/develop/release-management/features/storage/wipe-volume-using-blkdiscard.html.md

```diff
@@ -0,0 +1,84 @@
+---
+title: WipeVolumeUsingBlkdiscard
+authors: ishaby
+wiki_title: Features/WipeVolumeUsingBlkdiscard
+feature_name: Wipe volume using blkdiscard
+feature_modules: vdsm
+feature_status: Work in progress for oVirt 4.1
+---
+
+# Wipe volume using blkdiscard
+
+## Summary
+
+### How volumes are wiped today
+When a disk on a block domain has its ***Wipe After Delete*** property enabled and it is removed, vdsm first wipes its volumes (writes zeros on them) and only then removes them.<br/>
+Today, vdsm uses the `dd` linux command to wipe volumes.
+
+### What can be improved
+The problem with using `dd` to wipe volumes is that it is **very slow** (~ 7 minutes to wipe a 10GB volume on netapp in a simple development environment).<br/>
```

Comment:
Don't mention specific storage vendors. It would suffice to say that on storages that support it, blkdiscard is ten times faster.